messaging/src/test/java/org/axonframework/serialization/json/JacksonSerializerTest.java

```diff
@@ -103,7 +97,10 @@ void testSerializeAndDeserializeArray() {
 
     @Test
     void testSerializeAndDeserializeList() {
-        objectMapper.enableDefaultTyping(ObjectMapper.DefaultTyping.OBJECT_AND_NON_CONCRETE, JsonTypeInfo.As.PROPERTY);
+        objectMapper.activateDefaultTyping(
+                objectMapper.getPolymorphicTypeValidator(),
+                ObjectMapper.DefaultTyping.OBJECT_AND_NON_CONCRETE
```

Comment:
Shouldn't it be `NON_CONCRETE_AND_ARRAYS` (the same used on `JacksonSerializer`)?